Makefile

```diff
@@ -24,6 +25,11 @@ else    # $(MSBUILD) != 1
 _SLN_BUILD  = MSBUILD="$(MSBUILD)" tools/scripts/xabuild
 endif   # $(USE_MSBUILD) == 1
 
+ifneq ($(ANDROID_API),)
+BUILD_FRAMEWORK_VERSION=$(shell echo "$(ALL_FRAMEWORKS)"|tr -s " "|cut -d " " -s -f $(ANDROID_API))
+MSBUILD_FLAGS += /p:AndroidApiLevel=$(ANDROID_API) /p:AndroidFrameworkVersion=$(BUILD_FRAMEWORK_VERSION)
```

Comment:
...*actually*, the previous line should be removed, and this line should instead be:

```make
MSBUILD_FLAGS += /p:AndroidApiLevel=$(API_LEVEL) /p:AndroidFrameworkVersion=$(word $(API_LEVEL), $(ALL_FRAMEWORKS))
```